sources/Interop/Windows/um/dxcapi/IDxcAssembler.cs

```diff
@@ -0,0 +1,71 @@
+// Copyright © Tanner Gooding and Contributors. Licensed under the MIT License (MIT). See License.md in the repository root for more information.
+
+// Ported from DirectXShaderCompiler, include/dxc/dxcapi.h, at commit d356a8bc546da38612cac13d1e643de17a0ba38f
+// Original source is Copyright © Microsoft. All rights reserved.
+
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+namespace TerraFX.Interop
+{
+    public unsafe partial struct IDxcAssembler
```

Comment:
For each COM interface you should add a:
```
--with-attribute
IDxcAssembly=Guid("091F7A26-1C1F-4948-904B-E6E3A8A771D5")
...
```

And then to the `Windows.Manual.cs` add:
```csharp
public static readonly Guid IID_IDxcAssembler = new Guid(0x091F7A26, 0x1C1F, 0x4948, 0x90, 0x4B, 0xE6, 0xE3, 0xA8, 0xA7, 0x71, 0xD5);
```